sda-commons-server-prometheus/src/main/java/org/sdase/commons/server/prometheus/PrometheusBundle.java

```diff
@@ -95,8 +95,7 @@ private void createPrometheusRegistry(Environment environment) {
             onShutdown(
                 () -> {
                   Metrics.removeRegistry(meterRegistry);
-                  Metrics.globalRegistry.close();
-                  Metrics.globalRegistry.clear();
+                  CollectorRegistry.defaultRegistry.clear();
```

Comment:
`Metrics.globalRegistry.close();`
`Metrics.globalRegistry.clear();`

These codelines are handling micrometer registry, i would prefer to keep them

Reply:
Oh ok! I will add them back

Reply:
Micrometer test only complained about not having `Metrics.removeRegistry(meterRegistry);`, so I removed the other code

Reply:
updated :)